core/src/main/java/com/tweet/core/error/ErrorCode.java

```diff
@@ -2,7 +2,8 @@
 
 
 public enum ErrorCode {
-    USER_NOT_FOUND(404, "User with name '%s' not found"),
+    USER_NOT_FOUND(404, "The user with name [%s] is not found"),
+    TWEET_NOT_FOUND(404, "The tweet with id [%s] is not found"),
     GENERAL_ERROR(500, "Happened something wrong");
```

Comment:
"Something wrong has happened" Miss Yoda :P
But this looks like there is no control over flow, maybe less scary would be something like:
"Unexpected server behaviour", but up to you.

Reply:
Sure. will change it